src/e2e/app/cli/mysql/tests/tests/test_private_datasets.rs

```diff
@@ -27,7 +27,114 @@ kamu_cli_run_api_server_e2e_test!(
             content:
               users:
                 predefined:
-                  - accountName: kamu
+                  - accountName: admin
+                    isAdmin: true
+                  - accountName: alice
+                  - accountName: bob
+            "#
+        )),
+);
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+kamu_cli_run_api_server_e2e_test!(
+    storage = mysql,
+    fixture = kamu_cli_e2e_repo_tests::private_datasets::test_a_private_dataset_is_available_only_to_the_owner_or_admin_in_the_search_results,
+    // We need synthetic time for the tests, but the third-party JWT code
+    // uses the current time. Assuming that the token lifetime is 24 hours, we will
+    // use the projected date (the current day) as a workaround.
+    options = Options::default()
+        .with_multi_tenant()
+        .with_today_as_frozen_system_time()
+        .with_kamu_config(indoc::indoc!(
+            r#"
+            kind: CLIConfig
+            version: 1
+            content:
+              users:
+                predefined:
+                  - accountName: admin
+                    isAdmin: true
+                  - accountName: alice
+                  - accountName: bob
+            "#
+        )),
+);
```

Comment:
Note: these parts are repeating between all test incarnations.
I'm adding a required email property in a parallel branch, and editing 100 more copies does not make me happy.

Reply:
Yes, editing one constant would be more convenient

Extracted [the const]